sdv/single_table/copulas.py

```diff
@@ -0,0 +1,89 @@
+"""Wrappers around copulas models."""
+
+import copulas
+import copulas.multivariate
+import copulas.univariate
+
+from sdv.single_table.base import BaseSynthesizer
+
+
+class GaussianCopulaSynthesizer(BaseSynthesizer):
+    """Model wrapping ``copulas.multivariate.GaussianMultivariate`` copula.
+
+    Args:
+        metadata (sdv.metadata.SingleTableMetadata):
+            Single table metadata representing the data that this synthesizer will be used for.
+        enforce_min_max_values (bool):
+            Specify whether or not to clip the data returned by ``reverse_transform`` of
+            the numerical transformer, ``FloatFormatter``, to the min and max values seen
+            during ``fit``. Defaults to ``True``.
+        enforce_rounding (bool):
+            Define rounding scheme for ``numerical`` columns. If ``True``, the data returned
+            by ``reverse_transform`` will be rounded as in the original data. Defaults to ``True``.
+        numerical_distributions (dict):
+            Dictionary that maps field names from the table that is being modeled with
+            the distribution that needs to be used. The distributions can be passed as either
+            a ``copulas.univariate`` instance or as one of the following values:
+
+                * ``gaussian``: Use a Gaussian distribution.
+                * ``beta``: Use a Beta distribution.
+                * ``truncated_gaussian``: Use a Truncated Gaussian distribution.
+                * ``uniform``: Use a uniform distribution.
+                * ``gamma``: Use a Gamma distribution.
+                * ``gaussian_kde``: Use a GaussianKDE distribution. This model is non-parametric,
+                  so using this will make ``get_parameters`` unusable.
+
+        default_distribution (copulas.univariate.Univariate or str):
+            Copulas univariate distribution to use by default. Valid options are:
+
+                * ``gaussian``: Use a Gaussian distribution.
+                * ``beta``: Use a Beta distribution.
+                * ``truncated_gaussian``: Use a Truncated Gaussian distribution.
+                * ``uniform``: Use a uniform distribution.
+                * ``gamma``: Use a Gamma distribution.
+                * ``gaussian_kde``: Use a GaussianKDE distribution. This model is non-parametric,
+                  so using this will make ``get_parameters`` unusable.
+         Defaults to ``beta``.
+    """
+
+    _DISTRIBUTIONS = {
+        'gaussian': copulas.univariate.GaussianUnivariate,
+        'beta': copulas.univariate.BetaUnivariate,
+        'truncated_gaussian': copulas.univariate.TruncatedGaussian,
```

Comment:
can we change `gaussian` to `norm` and `truncated_gaussian` to `truncnorm` here and in the docstrings?